GiniVision/Classes/GiniImageDocument.swift

```diff
@@ -58,10 +57,7 @@ final public class GiniImageDocument: NSObject, GiniVisionDocument {
     
     func rotateImage(degrees:Int, imageOrientation:UIImageOrientation) {
         metaInformationManager.rotate(degrees: 90, imageOrientation: imageOrientation)
-        guard let data = metaInformationManager.imageByAddingMetadata() else {
-            assertionFailure("It wasn't possible to add metadata to the image")
```

Comment:
Why no assertion? Now it's even more likely that the call fails.

Reply:
Because PNG and GIF could be rotated even it doesn't have any effect in them.